### PR TITLE
allow ^0.14.7, tests pass

### PR DESCRIPTION
**Pull request by rharriso**
I reran the tests using `0.14.7`  (adjusted the dev dependencies) all passed.

### CODE REVIEW

**Comment by rharriso** [comment]
@mjackson any interest in this?

File: package.json [hunk 1]
```diff
@@ -42,8 +42,8 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^3.0.0",
     "pretty-bytes": "^4.0.2",
-    "react": "^15.3.0",
-    "react-dom": "^15.3.0",
+    "react": "^15.3.0 || ^0.14.7",
+    "react-dom":  "^15.3.0 || ^0.14.7",
```
Comment (mjackson) [inline package.json]: Please omit the extra space on this line.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,13 @@ The UMD build is also available on [unpkg](https://unpkg.com):
 
 You can find the library on `window.ReactMedia`.
 
+## Test
```
Comment (mjackson) [inline README.md]: There's no need to include testing instructions in the README.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 umd
+/node_modules/
```
Comment (mjackson) [inline .gitignore]: Please don't change our `.gitignore` :)

**Comment by mjackson** [comment]
@rharriso Wanna finish this up? Or should we close?

**Comment by rharriso** [comment]
@mjackson oh sorry, this got buried in my notifications.

**Comment by rharriso** [comment]
@mjackson updated from your feedback

**Comment by mjackson** [comment]
Thanks @rharriso! :D

**Comment by mjackson** [comment]
Released in version 1.4.0

**Comment by rharriso** [comment]
@mjackson thanks!

Also, I just realized I met your partner @ryanflorence at a training session last week. It was really enjoyable!

**Pull request merged**